scripts/index.js

```diff
@@ -1 +1,14 @@
-console.log(`There are ${characters.length} characters in the array.`);
\ No newline at end of file
+console.log(`There are ${characters.length} characters in the array.`);
+
+function howManyStartWith(letter){
+    results = 0
```

Comment:
Ouch! You'll want to use `let` or `const` to make sure you don't create globals.